README.md

```diff
@@ -95,8 +95,8 @@ The `last_processed_val` field represents the last processed element value, and
 |------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------------------------------------|
 | `url`            | String line for connection to Oracle.                                                                                                                               | **true** | `username/password@path:1521/my.domain.com` |
 | `table`          | The name of a table in the database that the connector should write to.                                                                                             | **true** | `users`                                     |
-| `keyColumn`      | Column name records should use for their `Key` fields.                                                                                                              | **true** | `id`                                        |
 | `orderingColumn` | Column name that the connector will use for ordering rows. Column must contain unique values and suitable for sorting, otherwise the snapshot won't work correctly. | **true** | `created_at`                                |
+| `keyColumn`      | Column name records should use for their `Key` fields.                                                                                                              | false    | `id`                                        |
```

Comment:
Column name  => Comma separated list of column names. And maybe change keyColumn to keyColumns

Reply:
Yeah, it's a mistake.